lib/solarwinds_apm/sampling/instrumentation_scope.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module SolarWindsAPM
+  module InstrumentationScope
+    @lock = Mutex.new
+    @version_cache = {}
+
+    ##
+    # add the gem library version to event
+    # p.s. gem library version is not same as the opentelemetry-instrumentation-* gem version
+    def self.gather_instrumented_framework(span_data)
+      scope_name = span_data.instrumentation_scope.name
+      scope_name = scope_name.downcase if scope_name
+      return unless scope_name&.include? 'opentelemetry::instrumentation'
+
+      framework = scope_name.split('::')[2..]&.join('::')
+      return if framework.nil? || framework.empty?
+
+      framework         = normalize_framework_name(framework)
+      framework_version = check_framework_version(framework)
+      SolarWindsAPM.logger.debug do
+        "[#{self.class}/#{__method__}] #{span_data.instrumentation_scope.name} with #{framework} and version #{framework_version}"
+      end
+      framework_version.nil? ? nil : { "Ruby.#{framework}.Version" => framework_version }
```

Comment:
I think we don't need to support this anymore, based on [recent conversation](https://swicloud.slack.com/archives/C2J43PE4R/p1743783678600129). @cheempz is that correct?

Reply:
That's correct! Can remove this file :)

Reply:
Removed.